saleor/checkout/base_calculations.py

```diff
@@ -0,0 +1,63 @@
+"""Contain functions which are base for calculating checkout properties.
+
+It's recommended to use functions from calculations.py module to take in account plugin
+manager.
+"""
+
+from typing import TYPE_CHECKING
+
+from prices import TaxedMoney
+
+from ..core.taxes import quantize_price, zero_taxed_money
+from ..extensions.manager import get_extensions_manager
+
+if TYPE_CHECKING:
+    from .models import Checkout, CheckoutLine
+    from ..discount.types import DiscountsListType
+
+
+def get_base_checkout_shipping_price(
+    checkout: "Checkout", discounts: "DiscountsListType" = None
+) -> TaxedMoney:
+    """Return checkout shipping price."""
+    # FIXME: should it take discounts in account?
```

Comment:
@korycins wdyt?

Reply:
I don't see any usage of discount for shipping, so I assume that we can skip them for base but plugin should still recieve a discounts for `calculate_checkout_shipping`